packages/cli/src/executable/AgenticaStart.ts

```diff
@@ -90,6 +89,189 @@ export namespace AgenticaStart {
       services,
     });
 
-    console.log(`🎉 Project ${projectName} created`);
+    console.log(`\n🎉 Project ${projectName} created`);
+
+    console.log(
+      `\n⚠️  ${chalk.yellow("Note:")} Please implement constructor values for each controller generated in agent.ts or index.ts`,
+    );
+  }
+
+  /**
+   * Get questions for `start` command.
+   */
+  const getQuestions = (
+    input: IAgenticaStart.IGetQuestionsInput,
+  ): QuestionCollection[] => {
+    const questions = [
+      {
+        type: "list",
+        name: "packageManager",
+        message: "Package Manager",
+        choices: [
+          "npm",
+          "pnpm",
+          {
+            name: `yarn (berry ${chalk.blueBright("is not supported")})`,
+            value: "yarn",
+          },
+        ],
+      },
+      input.options.project
+        ? null
+        : {
+            type: "list",
+            name: "projectType",
+            message: "Project Type",
+            choices: [
+              {
+                name: `NodeJS ${chalk.blueBright("Agent Server")}`,
+                value: "nodejs",
+              },
+              {
+                name: `NestJS ${chalk.blueBright("Agent Server")}`,
+                value: "nestjs",
+              },
+              {
+                name: `React ${chalk.blueBright("Client Application")}`,
+                value: "react",
+              },
+              {
+                name: `Standalone ${chalk.blueBright("Application")}`,
+                value: "standalone",
+              },
+            ] satisfies { name: string; value: ProjectOptionValue }[],
+          },
+      {
+        type: "checkbox",
+        name: "services",
+        message: "Embedded Controllers",
+        choices: input.services,
+      },
+      {
+        type: "input",
+        name: "openAIKey",
+        message: "Please enter your OPENAI_API_KEY:",
+      },
+    ] satisfies (QuestionCollection | null)[];
+
+    return questions.filter(
+      (question): question is Exclude<typeof question, null> =>
```

Comment:
```suggestion
      (question): question is NonNullable<typeof question> =>
```

or 

```suggestion
      (question) =>
```

this type is already inferred

Reply:
oh my gosh you genius